examples/index.html

```diff
@@ -392,6 +394,14 @@ <h1>Piling.js</h1>
                 >)
               </em>
               <em id="svg-credit">(Randomly generated)</em>
+              <em id="scatterplots-credit">
+                (From
+                <a
+                  href="https://www.kaggle.com/uciml/pima-indians-diabetes-database"
```

Comment:
Needs to be updated :)

Reply:
Oops I commented on some outdated code. The new code is also incorrect though. It says "Smith et al., 1988".